### ISSUE TITLE
Legiones Hereticus RoW

### ISSUE THREAD

**Issue by TheRavingRaven** [BSData/horus-heresy#2981]
Due to the wording of the rules within the Exemplary battles book I belive that legions heretics should have access to the normal and legion specific rites of war.

The book clearly states that Legiones Hereticus should be treated as legions astartes at army creation.

The caveat states that "unless otherwise specified, a unit with this special rule loses any benefits, options or wargear avaliable to their specific variant of the legiones astartes (x) special rule, and instead gains the benefits granted by their specific variant of the legiones hereticus (x) special rule instead" as this specifies UNIT and not DETATCHMENT, the detachment should be able to select rites of war as normal as long as it contains a master of the legion.

Thank you.

**Comment by Mayegelt** [comment]
Ok I have outlined this elsewhere, but can't find my original post about it 
I get where you are coming from with this one. And I find it frustrating the way it is.

However...
In the second paragraph it says
For the purposes of building an army, treat this special rule in the same manner as the Legiones Astartes (X)..."
Then says in the 3rd
"Unless specified overwise, a unit with this special rule loses any benefits, options or wargear..."
I know that is a bit paraphrased as I'm not going to re-type the whole page again, but those are the key bits here.

So that first part says you treat it in the same manner. This then goes on to talk about army list detachments and allies.
To be in the same manner, does not mean you treat it as if it is the original, but just treat it as if it were like the original.
The second bit says rather directly that you lose the options of the LA (X).)

On page 96 of LA or LH books in Rites of war it says...
"A detachment that includes at least one model with  both Legiones Astartes (X) special rule and Master of the Legion special rule..."  In the gaining access to Rites of War section.

![Screenshot_2023-11-07-07-11-26-531_com kobobooks android](https://github.com/BSData/horus-heresy/assets/20280410/a635079e-c824-4cef-ba0a-814fcfad5588)

![Screenshot_2023-11-07-07-23-07-089_com kobobooks android](https://github.com/BSData/horus-heresy/assets/20280410/0de832a1-cc34-4e53-b83a-a5b5dd5c8fa3


Hopefully an FAQ will come out to update it to allow it. But I also suspect that there might be a publication later that includes a bunch of universal corrupted RoW for LH (X) coming later.

I did however leave a hole that will allow you to use LH with RoW. If you select LA first, pick the RoW, then swap to LH after. It will say there is a validation error, but if your circle of players is playing differently to how myself, multiple podcasts I listen to, multiple groups I am part of, many event organisers have decided (it was about 95% consensus with the way I've implemented it, with about a 50/50 split of them saying it was dumb as well).
